src/overlays/actors/ovl_En_Bird/z_en_bird.c

```diff
@@ -86,19 +86,15 @@ void EnBird_Init(EnBird* this, GlobalContext* globalCtx) {
 void EnBird_Destroy(EnBird* this, GlobalContext* globalCtx) {
 }
 
-#ifdef NON_MATCHING
-// D_0600006C address is reused when it shouldn't be
-// also minor ordering differences
 void func_809C1CAC(EnBird* this, s16 params) {
     f32 frameCount = SkelAnime_GetFrameCount(&D_0600006C);
     f32 playbackSpeed = this->unk_19C ? 0.0f : 1.0f;
+    u32* anim = &D_0600006C;
```

Comment:
I was going to say we should probably use UNK_PTR for this type, though krim defined u32* as the type in this function in his open PR. u32 is probably fine cause its just a segment address, but I figured we would name it when we know exactly what kind of animation data it is

Reply:
That’s a good point. I was kind of just doing u32* as a general place holder but there’s no reason why we can’t declare these segment addresses as their actual pointers. I’ll update my PR accordingly.

Reply:
So do I need to make this UNK_PTR? Something else?